### PR TITLE
 convert python2 to python3

### PR DESCRIPTION
**Pull request by guohuang**

### CODE REVIEW

File: default/inputs.conf [hunk 1]
```diff
@@ -4,6 +4,6 @@ sourcetype = cisco:phishing
 message_limit = 1000
 interval    = 60
 duration = 5
-initial_start_date = 2019-12-01T00:00:00+00:00
+initial_start_date = 2022-04-01T00:00:00+00:00
```
Comment (micahkemp-splunk) [inline default/inputs.conf:7]: Should we include any value for `initial_start_date` in this public app? And if we do, what makes either of these values valid?

File: default/app.conf [hunk 2]
```diff
@@ -16,4 +16,4 @@ version = 1.0.2
 
 [package]
 id = TA-cisco_phishing
-check_for_updates = true
+check_for_updates = false
```
Comment (micahkemp-splunk) [inline default/app.conf:19]: I assume this app isn't on Splunkbase, which is why it shouldn't check for updates?

File: default/inputs.conf (hunk 1 above)
Comment (guohuang) [reply default/inputs.conf:7]: this is a default input, we need it to run, but customer can use local/inputs.conf to override it.

File: default/app.conf (hunk 2 above)
Comment (guohuang) [reply default/app.conf:19]: it is not on splunkbase

**Review by pham-man — APPROVED** [review]

File: default/inputs.conf (hunk 1 above)
Comment (micahkemp-splunk) [reply default/inputs.conf:7]: I can approve this as-is, but we shouldn't be in the business of including arbitrary defaults. If we update a default, it should have a meaning. If we don't have any valid defaults to provide, we could use the unix epoch.

**Review by micahkemp-splunk — APPROVED** [review]

**Pull request merged**